packages/nextjs/pages/builders/0x2F142f9F52af0845B7b8b1353E349Ca1Cb8EFE92.tsx

```diff
@@ -0,0 +1,92 @@
+import { useEffect, useState } from "react";
+import Link from "next/link";
+import { appChains } from "~~/services/web3/wagmiConnectors";
+
+export default function PapaWheelie() {
```

Comment:
Nitpick: could you follow the convention? In all the components, we are using a `const` and then exporting at the bottom of the file.

Reply:
sure thing, converted to name function expression and added the default export at the bottom.